### ISSUE TITLE
Create Second Round of GA4 'How-To' Resources

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#88133]
### Description
This task is to create "how-to" guides and other resources to help GA4 users navigate and use the new GA tool for the platform documentation site. Resources should work for those new to GA in general as well as for the existing UA users who are just looking to reproduce [as close as possible] what they did  in the old GA version. "How-to" guides should, at least initially, support the live trainings. 

-- Explore: Funnel Report -- Jamie

### Tasks

- [x] Each "how-to" and most other resource should include written step-by-step instructions with accompanying supportive visualizations (screenshots, animated gifs, infographic, etc)
- [x] Initial drafts shared with and reviewed by other A&I team members (someone who did not write it)
- [ ] Where possible, identify GA users from product teams to review and provide feedback
- [ ] ~Time and hosting solution permitting, pre-recorded videos of the how-to's will be added~

---

### Acceptance Criteria

- [x] Resources topics have been identified
- [x] Written resources have been created
- [x] Drafts have been shared with and reviewed by other A&I team members (someone who did not write it)

**Comment by johnny-jesensky-adhoc** [comment]
@Blivaditis, @jestutt -- Do y'all know what topic(s) you'd want to cover for the second round of Training How-Tos?

**Comment by Blivaditis** [comment]
This ticket will carry over into the next sprint, how to's are currently in progress.

**Comment by johnny-jesensky-adhoc** [comment]
@Blivaditis  -- could you post the link to the artifact or Confluence page where I can see the current progress on this?

**Comment by Blivaditis** [comment]
Just shared the google doc with you, will put it in confluence once the draft is complete.

**Comment by johnny-jesensky-adhoc** [comment]
Thanks for the update @Blivaditis  -- can you work out of Confluence in the meantime [here](https://vfs.atlassian.net/wiki/spaces/AT/pages/3290660872/GA4+Trainings+How-To+Resources)? This space in Confluence isn't available to VFS Teams, so it's better spot to work out of until we're ready to push to the Platform Website.

Linking the artifact here --> [How to: Sampling](https://docs.google.com/document/d/1I0j9JiUVcPo879fa2yNwgRzVrLHaUXawKhgDH6cAIxQ/edit).

@jwoodman5, @raywangoctova, FYI -- we made some progress on this, but this ultimately carried over into the following sprint.

**Comment by Blivaditis** [comment]
I just prefer the interface of docs, I moved over what I have so far to this [link](https://vfs.atlassian.net/wiki/spaces/AT/pages/3346923522/How+to+Sampling).

**Comment by johnny-jesensky-adhoc** [comment]
@Blivaditis -- just did a review, we might eventually want to get Jamie to review as well, though that'll have to wait until post-audit. Think this is an acceptable V1, so I'm going to move this to _**Done**_ for now.

Will likely create a separate ticket to account for getting these How-Tos published on the Platform Website.

FYI @jestutt
